internal/httputil/routing.go

```diff
@@ -15,7 +15,10 @@
 package httputil
 
 import (
+	"net/http"
 	"net/url"
+
+	"github.com/gorilla/mux"
```

Comment:
As a side note, do we have an issue to replace mux? We need to as it has been archived and isn't maintained anymore.

Reply:
Now we have https://github.com/matrix-org/dendrite/issues/3017